akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -98,6 +99,15 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
               }
             case _ ⇒
           }
+
+          // once captured, optional verbose logging of event
+          e match {
+            case _: SeenChanged ⇒ // ignore
+            case event ⇒
+              if (cluster.settings.LogInfoVerbose)
+                logInfo("Cluster Node [{}] - event {}", selfAddress, event)
```

Comment:
This is logging the prefix twice, since it's also added by `logInfo`.

```
Cluster Node [akka.tcp://ClusterLogSpec@192.168.2.174:63782] - Cluster Node [akka.tcp://ClusterLogSpec@192.168.2.174:63782] - event MemberUp(Member(address = akka.tcp://ClusterLogSpec@192.168.2.174:63782, status = Up))
```

should be `logInfo(" - event {}", selfAddress, event)`

Reply:
It is but the ticket specified 
```The log messages should follow the same format as other cluster logging, i.e. prefixed with "Cluster Node"``` 
:) removing.